index.html

```diff
@@ -2025,6 +2067,22 @@ <h5 id="Psionic Traits">Psionic Traits</h5>
                <td>Psionic Level 10
                </td>
             </tr>
+            <tr>
+               <td>Memory Walk
+               </td>
+               <td>
+                  You may touch a creature to walk through it's mind, glimpsing memories of its past. 
+                  You can direct what you see by asking to recall specific events.
+                  You and the creature are <a class="condition" data-tooltip-container>immobilised</a>
+                  for the duration of this trait.  You are free to end this at any time.
```

Comment:
Also nice, if a creature doesn't resist this, can you basically jsut walk through freely? Just mention what you wanna see and you can delve into those memories? I mildly worry about how strong that could be from an RP/GM perspective, but I'm cool with it for now. Again you should be essentially unconscious but not prone and maybe aware of little things like hearing and touch.